css/index.css

```diff
@@ -1 +1,551 @@
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 100%;
+  max-width: 880px;
+  margin: 0 auto;
+}
+
+.container header {
+    width: 100%;
+}
+
+.container .main-content {
+    width: 100%;
+}
+
+
+
+
+
+header {
+	display: flex;
+    justify-content: space-between;
+    
+
+
+}
+
+
+header nav {
+  width: 100%;
+  margin-top: 5%;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 10%
+   
+}
+
+header nav a {
+  font-size: .90rem;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: flex;
+  margin-top: 10%;
+  padding-right: 5%
+  
+}
+
+
+
+.cta {
+  
+  display: flex;
+  flex-direction: row;
+}
+
+
+
+.cta .cta-text {
+  display: flex;
+  flex-wrap: wrap;
+  vertical-align: top;
+  font-size: 4.8125rem;
+  margin-top: 20%;
+  text-align: center;
+  padding: 0px 5%;
+ 
+  width: 100%;
+  justify-content: center;
+}
+
+.cta .cta-text button {
+  display: flex;
+  
+  background: white;
+  vertical-align: top;
+  margin-top: 5%;
+  padding: 5% 20%;
+  font-size: 1rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+  margin-top: 69px;
+  
+  
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 10%;
+  padding-top: 10%;
+}
+
+.main-content .top-content {
+  padding: 0 1%;
+  display: flex;
+  margin-top: 5%;
+  flex-wrap: wrap;
+
+ 
+}
+.main-content .top-content .text-content {
+   
+    max-width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+    
+} 
+
+.main-content .top-content .text-content1 {
+    
+    max-width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+    
+} 
+
+.main-content .bottom-content {
+    
+    padding: 0 5%px;
+    display: flex;
+    margin-top: 5%;
+    flex-wrap: wrap;
+
+}
+
+
+
+.bottom-content .text-content2 {
+   
+    max-width: 33%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+}
+
+
+.bottom-content .text-content3 {
+   
+    max-width: 33%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+}
+
+
+.bottom-content .text-content4 {
+    
+    max-width: 33%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5%;
+}
+
+
+
+
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+  
+}
+
+.main-content .middle-img {
+  margin-top: 10%;
+  max-width: 100%;
+}
+
+.main-content .bottom-content .text-content {
```

Comment:
It looks like you have some repeating sections with different styles. You can combine these to clean it up.